tests/ui/test_non_admin_user.py

```diff
@@ -74,6 +77,7 @@ class TestUnprivilegedUserODFAccess(E2ETest):
     @skipif_ibm_cloud_managed
```

Comment:
Curious to know why do we skip it on IBM?

Reply:
@am-agrawa that was added as part of PR https://github.com/red-hat-storage/ocs-ci/pull/7122